src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -519,6 +519,13 @@ EMSCRIPTEN_KEEPALIVE void mono_wasm_load_profiler_aot (const char *desc) { mono_
       <_WasmAOTSearchPaths Include="$(MicrosoftNetCoreAppRuntimePackRidDir)\lib\net7.0" />
     </ItemGroup>
 
+    <ItemGroup>
+      <!-- Skip known-huge assemblies specified in csproj -->
+      <_AotInputAssemblies Condition="'%(Filename)' != '' and '@(_AotInputAssemblies->Metadata(`Filename`))' != '' and '@(HighAotMemoryUsageAssembly->Metadata(`Filename`))' != ''">
```

Comment:
This file should use an item like `@(_AOT_InternalForceInterpretAssemblies)`, and we can add `@(HighAotMemoryUsageAssembly)` to this item in the eng/testing targets.